game/player.asm

```diff
@@ -0,0 +1,145 @@
+.section "Player" BANK 0 SLOT "ROM"
+
+nop
+
+.struct Player
+    oam_manager_ptr instanceof OAMManager
```

Comment:
This shouldn't be an instanceof OAMManager. It should just be a `dw`-length pointer that gets set. See comment in game.asm.